src/back/routes/vestings.ts

```diff
@@ -7,9 +7,14 @@ import { VestingService } from '../../services/VestingService'
 import { validateAddress } from '../utils/validations'
 
 export default routes((router) => {
+  router.get('/all-vestings', handleAPI(getAllVestings))
```

Comment:
Just `vestings`? We don't have `all-projects` or `all-proposals`

Reply:
Will be removed in the next PR